_docs/resources/sketch-library.md

```diff
@@ -1,263 +1,36 @@
-
-1. Open Sketch and click **Sketch > Preferences**.
-
-1. On the **Preferences** dialog box, click **Libraries**, and then click **Add Library**.
-
-1. Click **Google Drive > sketch_libraries > helix-ui-components.sketch**.
-
-1. Click **Open**.
-
-
-{% endcolumn %}
-
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-
-   {% figure [caption:""] [class:"image bg-light border"] %}
-   <embed src="{{site.url}}/assets/images/install-sketch-library/Add-library-1.png"/>
-   {% endfigure %}
-
-   {% figure [caption:""] [class:"image bg-light border"] %}
-   <embed src="{{site.url}}/assets/images/install-sketch-library/Add-library-2.png"/>
-   {% endfigure %}
-
-   {% figure [caption:"Sketch adds the Helix library to the list of libraries."] [class:"image bg-light border"] %}
-   <embed src="{{site.url}}/assets/images/install-sketch-library/Add-library-3.png"/>
-   {% endfigure %}
-
-
-{% endcolumn %}
-
-</div>
-</section>
-
-<section class="static-section" markdown="1">
-
-### Option 2: Download directly from GitHub
-
-<div class="hxRow" markdown="1">
-
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
-
-If you do not want to use the Google Drive approach to installing the Sketch Library, you can download the library to your local machine and then import it into Sketch.
-
-{% include note.html content="When you import the Sketch library from your local machine, you do not receive automatic updates." %}
-
-{% endcolumn %}
-
-</div>
-</section>
-
-<section class="static-section" markdown="1">
-#### Task 1: Download the Sketch Library from Github
-
-<div class="hxRow" markdown="1">
-
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
-
-1. Go to the [hxSketchLibrary repository](https://github.com/technabors/hxSketchLibrary).
-
-1. Select the **helix-ui-components.sketch** file from the repo.
-
-1. Download Sketch Library to your local machine.
-
-    {% include note.html content="The repo currently houses a separate library for creating redlines and the team plans to add more libraries to this repo in the future. Feel free to **star**, **watch**, and contribute!" %}
-
-{% endcolumn %}
-
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
-
-   {% figure [caption:""] [class:"image bg-light border"] %}
-   <embed src="{{site.url}}/assets/images/install-sketch-library/download-from-github.png"/>
-   {% endfigure %}
-
-{% endcolumn %}
-
-</div>
-</section>
-
-#### Task 2: Import the Helix Sketch Library into Sketch
-
-<section class="static-section" markdown="1">
-<div class="hxRow" markdown="1">
-
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-Complete the following steps to bring the Helix Sketch Library into Sketch.
-
-1. Open Sketch and click **Sketch > Preferences**.
+1. After downloading the file, move it somewhere easy to remember such as your desktop.
```

Comment:
After you download the Sketch file, move it somewhere that you can remember, such as your desktop.